builds/misc/images.yaml

```diff
@@ -431,7 +272,7 @@ jobs:
           imageName: azureiotedge-analyzer
           project: TestAnalyzer
           
-      # Functions Sample
+      # Functions Sample - Not supported on arm64
```

Comment:
do you need to update this comment? since we only build amd64 for Windows now.

Reply:
This comment have been for the linux case. I will move.